pkg/api/conversion.go

```diff
@@ -234,6 +234,9 @@ func Convert_resource_Quantity_To_resource_Quantity(in *resource.Quantity, out *
 }
 
 func Convert_map_to_unversioned_LabelSelector(in *map[string]string, out *unversioned.LabelSelector, s conversion.Scope) error {
+	if out == nil {
```

Comment:
hrm, yes but since we are adding fields into `out` as you passed in as reference, resetting it might not be expected?